package.json

```diff
@@ -2,13 +2,21 @@
   "name": "blurple-hammer",
   "main": "src/bot.js",
   "scripts": {
-    "start": "node ."
+    "start": "node .",
+    "lint": "eslint --ext \".js\" ./"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/project-blurple/blurple-hammer.git"
   },
   "author": "Promise <me@promise.solutions> (https://promise.solutions/)",
+  "contributors": [
+    "Promise <me@promise.solutions> (https://promise.solutions/)",
+    "Gigabitzz <> (https://www.gigabitz.pw)",
+    "wrac4242 <> ()",
+    "flamableassassin <lighter@highlyflammable.tech> (https://highlyflammable.tech/)"
+   ],
+
```

Comment:
Contributors are shown here: https://github.com/project-blurple/blurple-hammer/graphs/contributors
Will not keep this up-to-date. Please remove before I can accept the PR.

Reply:
👍 i shall remove it